spec/requests/account_activations_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'Account Activations', type: :request do
+  describe 'GET root_url' do 
+    let!(:user) { User.create(name: 'Sunzuiana Alexandra',
```

Comment:
The params here seem to not be aligned correctly. Also, the FactoryBot observation also applies here